src/contractTest/java/uk/gov/hmcts/reform/sscs/consumer/GetHearingPactConsumerTest.java

```diff
@@ -0,0 +1,309 @@
+import org.springframework.cloud.openfeign.EnableFeignClients;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.sscs.ContractTestDataProvider;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.HearingGetResponse;
+import uk.gov.hmcts.reform.sscs.service.HmcHearingApi;
+
+import java.time.LocalDateTime;
+import java.util.concurrent.TimeUnit;
+
+@SpringBootTest
+@ExtendWith(PactConsumerTestExt.class)
+@EnableFeignClients(basePackages = {"uk.gov.hmcts.reform.sscs.service"})
+@ActiveProfiles("contract")
+@PactTestFor(port = "10000")
+public class GetHearingPactConsumerTest extends ContractTestDataProvider {
+
+    private static final String PATH_HEARING = "/hearing";
+    private static final String FIELD_ID = "id";
+    private static final String VALID_CASE_ID = "123";
+    private static final String OPTION_FIELD_IS_VALID = "?isValid";
+    private static final String VALID_NO_CONTENT_CASE_ID = "0";
+
+    private static final String BAD_REQUEST_CASE_ID = "400";
+    private static final String UNAUTHORISED_CASE_ID = "401";
+    private static final String FORBIDDEN_CASE_ID = "403";
+    private static final String NOT_FOUND_CASE_ID = "404";
+
+    private static final LocalDateTime date = LocalDateTime.now();
+
+    @Autowired
+    private HmcHearingApi hmcHearingApi;
+
+    @BeforeEach
+    public void timeGapBetweenEachTest() throws InterruptedException {
+        TimeUnit.SECONDS.sleep(2);
+    }
+
+
+    @Pact(consumer = CONSUMER_NAME)
+    public RequestResponsePact getHearing(PactDslWithProvider builder) {
+
+        return builder
+            .given("sscs hearing api successfully returns case")
+            .uponReceiving("Request to GET hearing for given valid case id")
+            .path(PATH_HEARING)
+            .method(HttpMethod.GET.toString())
+            .query(FIELD_ID + "=" + VALID_CASE_ID)
+            .headers(headers)
+            .willRespondWith()
+            .status(HttpStatus.OK.value())
+            .body(generateValidHearingGetResponsePactDslJsonBody(date))
+            .toPact();
+    }
+
+    @Test
+    @PactTestFor(pactMethod = "getHearing")
+    public void shouldSuccessfullyGetHearing() throws Exception {
+
+        HearingGetResponse result = hmcHearingApi.getHearingRequest(
+            IDAM_OAUTH2_TOKEN,
+            SERVICE_AUTHORIZATION_TOKEN,
+            VALID_CASE_ID
+        );
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JavaTimeModule());
+        objectMapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
+        HearingGetResponse expected = objectMapper.readValue(
+            generateValidHearingGetResponsePactDslJsonBody(date).toString(),
+            HearingGetResponse.class
+        );
+
+        Assertions.assertEquals(expected, result);
```

Comment:
Very Minor: static import assertions and remove new lines underneath please - same applies to other tests